nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -269,15 +269,21 @@ def _taxonomy_index_path(self, name, lang, taxonomy):
         path, append_index, _ = self._parse_path_result(result)
         return self._postprocess_path(path, lang, append_index=append_index, dest_type='list')
 
-    def _taxonomy_path(self, name, lang, taxonomy, dest_type='page'):
+    def _taxonomy_path(self, name, lang, taxonomy, dest_type='page', page=None):
```

Comment:
Ah, and in line 247 it should be `dest_type in ('page', 'feed')` and not `dest_type == 'page'`.